website_crm_privacy_policy/templates/contactus.xml

```diff
@@ -0,0 +1,31 @@
+<odoo>
+    <!-- Privacy Policy -->
+    <template
+        id="contactus_form"
+        inherit_id="website_crm.contactus_form"
+        name="Add privacy policy checkbox"
+    >
+        <xpath
+            expr="//form//div/div/span[@id='s_website_form_result']/.."
+            position="before"
+        >
+            <div class="form-group row form-field s_website_form_required_custom">
+                <div class="col-sm" style="margin-left: 200px;">
+                    <div class="form-check">
+                        <input
+                            type="checkbox"
+                            value="accepted"
+                            id="privacy_policy"
+                            name="privacy_policy"
+                            class="s_website_form_input form-check-input"
+                            required="required"
+                        />
+                        <label class="form-check-label" for="privacy_policy">
+                            <t t-call="website_legal_page.acceptance_full" />
+                        </label>
+                    </div>
+                </div>
+            </div>
```

Comment:
it looks acceptable to me putting it like this:
```suggestion
          <div class="form-group col-12 s_website_form_field s_website_form_required_custom">
                <div class="form-check">
                    <input
                        type="checkbox"
                        value="accepted"
                        id="privacy_policy"
                        name="privacy_policy"
                        class="s_website_form_input form-check-input"
                        required="required"
                    />
                    <label class="form-check-label" for="privacy_policy">
                        <t t-call="website_legal_page.acceptance_full" />
                    </label>
                </div>
            </div>
```

Reply:
Thank you very much, it is perfect now with this change.